### DISCUSSION TITLE
multi branch workflow - how to fly the tracker.yml?

### DISCUSSION THREAD

**Discussion by fong1999** [concourse/concourse#8726] [General]
Hi! I am trying to use concourse multi-branch workflow concept. I am reading the document here - https://concourse-ci.org/multi-branch-workflows.html. But not sure how to fly the tracker.yml.

After I created `tracker.yml` file, I should be able to fly it or? How do I get is run and create a pipeline per feature branch? Does it do it automatically? Does it get created one time, then it will monitor our github branches? 

I tried below command, but getting error `error: unknown flag enable-across-setp`

`fly sp -c ./ci/multi-branch/tracker.yml -p tracker-pipeline --enable-across-step`

What am I missing? Thanks in advance!

**Comment by jriddy** [comment]
You need to [enable across steps](https://concourse-ci.org/across-step.html) with `CONCOURSE_ENABLE_ACROSS_STEP=true`
